internal/docker/supported.go

```diff
@@ -83,3 +86,19 @@ func cleanupDockerResource(pool *dockertest.Pool, resource *dockertest.Resource)
 	}
 	return fmt.Errorf("Failed to cleanup local container: %s", err)
 }
+
+// splitImage takes the WithDatabaseImage option and separates
+// it into repo + tag. If a tag is not found, it sets the tag to latest
+func splitImage(opt ...Option) (string, string) {
+	opts := GetOpts(opt...)
+	if opts.withContainerImage == "" {
+		return "postgres", "latest"
+	}
+	separated := strings.Split(opts.withContainerImage, ":")
+	if len(separated) > 1 {
```

Comment:
Docker image tags cannot themselves contain colons:
```
$ docker tag postgres:11 mypostgres:11:foo
Error parsing reference: "mypostgres:11:foo" is not a valid repository/tag: invalid reference format
```
So this function should really return an error as well, and you could put this logic into a `switch` on the length of the split strings, with cases for 1, 2, and a default that errors.